wavefront/resource_alert.go

```diff
@@ -188,3 +216,36 @@ func resourceAlertDelete(d *schema.ResourceData, m interface{}) error {
 	d.SetId("")
 	return nil
 }
+
+func validateAlertConditions(a *wavefront.Alert, d *schema.ResourceData) error {
```

Comment:
Validate that the threshold_targets and threshold_conditions are valid - want, severe, info